thirdparty/openapi3/schemas/AccountAddress.yaml

```diff
@@ -1,3 +1,4 @@
+title: AccountAddress
```

Comment:
We have used `AccountId and `AccountAddress` interchangeably through out our services.

In reference to the original proposed specification change https://github.com/mojaloop/mojaloop-specification/issues/61 and https://github.com/mojaloop/account-lookup-service/commit/bdfa2117d8bb67aacb7740304dcd1eeb6e9d0ed1
which is my initial implementation of the proposed change in the ALS.

AccountAddress should be the correct name.

My question is do we align things like in example `sourceAccountId` in `ThirdpartyTransactionRequest` to be `sourceAccountId` here and throughout our services.

Discussion is open.